src/Codeception/Lib/Connector/Yii2.php

```diff
@@ -79,7 +81,7 @@ class Yii2 extends Client
     /**
      * @return \yii\web\Application
      */
-    public function getApplication()
+    protected function getApplication()
```

Comment:
Wait, this is a breaking change, the method was not declared as `@internal`, this can't be push in a minor release

Reply:
Good catch!

Reply:
@SamMousa please review https://github.com/Codeception/Codeception/pull/5202

Reply:
I catch it %(

Error: Call to protected method Codeception\Lib\Connector\Yii2::getApplication() from context 'bscheshirwork\Codeception\Module\DbYii2Config' in /repo/vendor/bscheshirwork/codeception-db-yii2-config/DbYii2Config.php on line 41